nostr/relay.py

```diff
@@ -115,21 +125,18 @@ def to_json_object(self) -> dict:
         }
 
     def _on_open(self, class_obj):
-        self.connected = True
+        pass
 
     def _on_close(self, class_obj, status_code, message):
-        self.connected = False
+        self.error_counter = 0
 
     def _on_message(self, class_obj, message: str):
         self.message_pool.add_message(message, self.url)
     
     def _on_error(self, class_obj, error):
-        self.connected = False
```

Comment:
I don't understand why this is removed. Errors are thrown when the relay disconnects the client. This is not a nostr error but a WebSocket error. I haven't encountered a case where an error was not a disconnect.